person.rb

```diff
@@ -4,10 +4,10 @@
 require_relative 'trimmer_decorator'
 require_relative 'rental'
 class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id, :rentals
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age:, name: 'Unknown', parent_permission: true)
     super()
     @id = SecureRandom.uuid
```

Comment:
- [ ] As shown in the [reference video](0bd85d17-925a-4c67-9a88-482925d5ef78) your ID should be a number not a string, currently with this line of code you are generating strings as ID, please kindly add the necessary corrections :+1: